### PR TITLE
test: add missing _c_unused_

### PR DESCRIPTION
**Pull request by mdionisio**

### CODE REVIEW

File: src/test-api.c [hunk 1]
```diff
@@ -41,7 +42,7 @@ static void test_api(void) {
         free(argv);
 }
 
-int main(int argc, char **argv) {
+int main(_c_unused_ int argc, _c_unused_ char **argv) {
```
Comment (dvdhrm) [inline src/test-api.c:45]: Can we turn this into `int main(void) {` instead? This avoids the dependency and makes it easier to read, doesn't it?

**Review by dvdhrm — CHANGES_REQUESTED** [review]
Thanks for the PR! One comment inline.

File: src/test-api.c (hunk 1 above)
Comment (mdionisio) [reply src/test-api.c:45]: Do you think it is better `int main(void)` ???  I think not, `int main(void)` is not valid in c++ compiler from where it is better to do `int main()`. The dependency problems cames in meson file from where is uses `link_with` and not `dependencies` like for other test.

Comment (dvdhrm) [reply src/test-api.c:45]: `int main(void) { }` is valid C, isn't it? I think it is preferable to `unused` annotations.

Also note that this was already fixed by a PR yesterday. Can you check whether #3 solves this for you?

**Pull request closed without merging**